### PR TITLE
Instant Client comment and doc updates

### PR DESCRIPTION
**Pull request by cjbj**
"Docker" -> "Container" changes, as hinted at on Slack.

### CODE REVIEW

File: OracleInstantClient/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# About this Docker Image
+# About this Container Image
```
Comment (Djelibeybi) [inline OracleInstantClient/README.md:1]: Neither container nor image are proper nouns so neither of them should be capitalised.

File: OracleInstantClient/dockerfiles/12.2.0.1/Dockerfile [hunk 2]
```diff
@@ -1,11 +1,9 @@
 # LICENSE UPL 1.0
 #
-# Copyright (c) 2014-2015 Oracle and/or its affiliates. All rights reserved.
+# Copyright (c) 2014, 2020 Oracle and/or its affiliates. All rights reserved.
```
Comment (Djelibeybi) [inline OracleInstantClient/dockerfiles/12.2.0.1/Dockerfile:3]: If you're changing copyright notices anyway, please remove the "All rights reserved." phrase as well. We no longer include that phrase.

File: OracleInstantClient/dockerfiles/18/Dockerfile [hunk 3]
```diff
@@ -1,11 +1,8 @@
 # LICENSE UPL 1.0
 #
-# Copyright (c) 2014, 2019, Oracle and/or its affiliates. All rights reserved.
+# Copyright (c) 2014, 2020, Oracle and/or its affiliates. All rights reserved.
```
Comment (Djelibeybi) [inline OracleInstantClient/dockerfiles/18/Dockerfile:3]: If you're changing copyright notices anyway, please remove the "All rights reserved." phrase as well. We no longer include that phrase.

**Review by Djelibeybi — CHANGES_REQUESTED** [review]
Considering this is purely a documentation update, I went into full copy editor mode. :)

File: OracleInstantClient/README.md (hunk 1 above)
Comment (cjbj) [reply OracleInstantClient/README.md:1]: You're supposed to be away aren't you? It's a heading IMHO.  Hence the '#'

Comment (Djelibeybi) [reply OracleInstantClient/README.md:1]: Oracle standard is sentence case for headings which means only the first word is capitalised.

Comment (cjbj) [reply OracleInstantClient/README.md:1]: See Slack

**Review by Djelibeybi — APPROVED** [review]

**Pull request merged**